app/models/user.rb

```diff
@@ -9,6 +9,7 @@ class User < ApplicationRecord
 
   has_many :signed_agreements
   has_many :carts
+<<<<<<< HEAD
```

Comment:
Look like you've still got merge conflict stuff lingering in there?

Reply:
Fixed!